docs/sources/tempo/operations/dedicated_columns.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Dedicated columns
+description: Learn how to use dedicated columns to improve query performance.
+weight: 42
+---
+
+# Dedicated columns
+
```

Comment:
Are we releasing Parquetv3 as experimental? If so, what do you think about adding this admonition to the doc? This is standard wording that we use for experimental features/procedures. 

```suggestion

{{% admonition type="warning" %}}
Dedicated columns is part of Parquetv3, an [experimental feature](/docs/release-life-cycle/). Engineering and on-call support is not available. Documentation is either limited or not provided outside of code comments. No SLA is provided.  
{{% /admonition %}}

```

Reply:
No, not really. It's the first iteration of making the schema more dynamic, but it's not experimental I'd say.

Reply:
OKay. I can remove the suggestion to add the admonition.